packages/main/src/ComboBox.ts

```diff
@@ -364,29 +350,35 @@ class ComboBox extends UI5Element {
 	/**
 	 * Receives id(or many ids) of the elements that label the component
 	 * @type {string}
-	 * @name sap.ui.webc.main.ComboBox.prototype.accessibleNameRef
-	 * @defaultvalue ""
+	 * @default ""
 	 * @public
 	 * @since 1.0.0-rc.15
 	 */
 	@property()
 	accessibleNameRef!: string;
 
+	/**
+	 * @private
```

Comment:
When we specify the privacy and add documentation, I expect to add description of the property as well. I know that it is not part of your task, but we can discuss it with the team

Reply:
Discussed with the team. We agreed that description is good to be added, but will be done in a separate change